action.cjs

```diff
@@ -0,0 +1,221 @@
+const fs = require('fs')
+
+function hashFiles (filename) {
```

Comment:
nit: prefer to remove this definition (since it's only one line) or rename to something like `existsFile` since it's not a hash and only takes one file as input

Reply:
imports too?